### PR TITLE
INT-2595 Dockerfile now creates an image without the git.

### PR DESCRIPTION
**Pull request by aliculPix4D**
Removed unnecessary files.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,6 +16,4 @@ RUN apk-install git ca-certificates libressl-dev \
     && rm -vf /var/cache/apk/* \
     && gem install /tmp/concourse-github-status*.gem
 
-WORKDIR /opt/resource
-
-RUN find $(gem environment gemdirs) -type f -path '*/concourse-github-status-*/bin/*' -exec ln -s '{}' \;
+RUN find $(gem environment gemdirs) -type f -path '*/concourse-github-status-*/bin/*' -exec ln -s '{}' \;
```
Comment (marco-m) [inline Dockerfile]: @aliculPix4D I think your editor is configured to remove the newline on the last line of the file; could you please fix your editor configuration, we want to keep the newline?

File: concourse-github-status.gemspec [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
+  gem.files         = ["bin/check", "bin/in", "bin/out", "concourse-github-status.gemspec", "lib/github-status.rb", "lib/github-status/check.rb", "lib/github-status/core.rb", "lib/github-status/in.rb", "lib/github-status/out.rb", "lib/github-status/support/git.rb", "lib/github-status/support/github.rb", "lib/github-status/support/params.rb", "lib/github-status/support/source.rb", "lib/github-status/version.rb"]
+  gem.executables   = ["check", "in", "out"]
 
   gem.require_paths = ['lib']
 
   gem.add_runtime_dependency 'concourse-fuselage', '~> 0.1', '>= 0.1.0'
   gem.add_runtime_dependency 'git',                '~> 1.3', '>= 1.3.0'
   gem.add_runtime_dependency 'octokit',            '~> 4.2', '>= 4.2.0'
-end
+end
```
Comment (marco-m) [inline concourse-github-status.gemspec]: same as above

File: Gemfile.lock [hunk 3]
```diff
@@ -1,33 +0,0 @@
-PATH
```
Comment (marco-m) [inline Gemfile.lock]: Are you sure ? Although I am not familiar with Ruby, the lockfile is normally needed, to ensure dependencies do not change. Or am I missing something ?

I am pinging @david-sinuela-pix4d-dev here as Ruby expert :-)

File: concourse-github-status.gemspec [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
+  gem.files         = ["bin/check", "bin/in", "bin/out", "concourse-github-status.gemspec", "lib/github-status.rb", "lib/github-status/check.rb", "lib/github-status/core.rb", "lib/github-status/in.rb", "lib/github-status/out.rb", "lib/github-status/support/git.rb", "lib/github-status/support/github.rb", "lib/github-status/support/params.rb", "lib/github-status/support/source.rb", "lib/github-status/version.rb"]
+  gem.executables   = ["check", "in", "out"]
 
   gem.require_paths = ['lib']
 
   gem.add_runtime_dependency 'concourse-fuselage', '~> 0.1', '>= 0.1.0'
   gem.add_runtime_dependency 'git',                '~> 1.3', '>= 1.3.0'
   gem.add_runtime_dependency 'octokit',            '~> 4.2', '>= 4.2.0'
 end
+gem.files         = `git ls-files -z`.split("\x0")
```
Comment (aledeganopix4d) [inline concourse-github-status.gemspec]: I think this block wasn't meant to be re-inserted.

**Pull request merged**